ansible/all-in-one/docker/stack.yml.j2

```diff
@@ -30,6 +30,7 @@ services:
     image: {{ xsnippet_web_backend_image }}
     environment:
       - XSNIPPET_API_URL=http://{{ xsnippet_api_server_name }}
+      - ACCESS_LOG_FORMAT='%t %{X-Real-IP}i "%r" %s %b %{User-Agent}i" %Tf'
```

Comment:
Isn't it a default value? Do we really want to override it if default is good here?

Reply:
@ikalnytskyi So there is a minor difference here: xsnippet-api defaults to using of "remote addr", as if it's deployed without reverse proxy, and here we specifically override that to use the headers provided by our nginx config.

Reply:
Ah, got it!